#raw servo positions from crustcrawler arm

float32[6] positions
